.github/scripts/update_commit_hashes.py

```diff
@@ -115,9 +159,15 @@ def main() -> None:
             # no existing pr, so make a new one and approve it
             pr_num = make_pr(args.repo_name, branch_name)
             approve_pr(pr_num)
-    if pr_num is not None:
         # comment to merge if all checks are green
         make_comment(pr_num)
+    else:
+        print(
+            f"tried to update from old hash: {old_hash} to new hash: {hash} but the old hash seems to be newer, not creating pr"
```

Comment:
Btw, could we have this print statement as a comment somehow? It would be useful to see this instead of having a closed PR without seeing any reason I guess

Reply:
Ah wait, I think there won't even be any PR in the first place. Never mind :)